extensions/test-src/org/pentaho/platform/plugin/services/importexport/exportManifest/bindings/EntityMetaDataTest.java

```diff
@@ -1,33 +1,30 @@
 /*
- * ******************************************************************************
+ * This program is free software; you can redistribute it and/or modify it under the
+ * terms of the GNU General Public License, version 2 as published by the Free Software
+ * Foundation.
  *
- * Copyright (C) 2002-2015 by Pentaho : http://www.pentaho.com
+ * You should have received a copy of the GNU General Public License along with this
+ * program; if not, you can obtain a copy at http://www.gnu.org/licenses/gpl-2.0.html
```

Comment:
@AliaksandrDrebenchuk please use the proper license header for extensions project